site/rebase-image.md

```diff
@@ -12,6 +12,7 @@ Usage: imagetool rebase [OPTIONS]
 | --- | --- | --- |
 | `--additionalBuildCommands` | Path to a file with additional build commands. For more details, see [Additional information](#additional-information). |
 | `--additionalBuildFiles` | Additional files that are required by your `additionalBuildCommands`.  A comma separated list of files that should be copied to the build context. |
+| `--buildNetwork` | Networking mode for the RUN instructions during the image build.  See Docker --network.  |   |
```

Comment:
If Docker --network is an attribute of a Docker command, it should be written as `docker --network` (if docker is part of the command) or Docker `--network` (if `network` if it is an attribute of some other Docker command).